.github/workflows/ci-push.yml

```diff
@@ -226,7 +226,26 @@ jobs:
           raco pkg config --set catalogs $PWD/rktcat/ https://pkgs.racket-lang.org https://planet-compats.racket-lang.org
           raco pkg install --auto racket-test
       - name: Run tests
-        run: raco test -l tests/racket/test
+        run: |
+          raco test -l tests/racket/test
+          racket -l tests/racket/contract/all
+          raco test -l tests/json/json
+          raco test -l tests/file/main
+          raco test -l tests/net/head
+          raco test -l tests/net/uri-codec
+          raco test -l tests/net/url
```

Comment:
I thought we needed to install "net-test" as well.

Reply:
Hummm, I assume not since the test run did not fail but I will check.

Reply:
It seems net-test is not needed to install as a package because it's installed as a dependency of `racket-test` installed earlier:
```
Downloading repository git://github.com/racket/string-constants/?path=string-constants-lib
The following uninstalled packages were listed as dependencies
and they were automatically installed:
 dependencies of racket-test:
   compiler-lib
   sandbox-lib
   compatibility-lib
   eli-tester
   planet-lib
   net-lib
   net-test
...
```